src/applications/proxy-rewrite/partials/mobile/header/mega-menu/level-two-links.js

```diff
@@ -0,0 +1,103 @@
+import { kebabCase } from 'lodash';
```

Comment:
Plain HTML version of the mega menu (used for mobile). Includes the things you see when you click on a top-level mega menu dropdown e.g. benefit hub links and the child links for About VA.